cli/cmd/event.go

```diff
@@ -0,0 +1,47 @@
+// (c) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package cmd
+
+import (
+	teleportermessenger "github.com/ava-labs/teleporter/abi-bindings/go/Teleporter/TeleporterMessenger"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+)
+
+var (
+	topicArgs []string
+	data      []byte
+)
+
+var eventCmd = &cobra.Command{
+	Use:   "event --topics topic1, topic2",
```

Comment:
From the usage string, it's not clear how to provide both topics and data. Is it
`--topics topic1 topic2 -- --data data`? Or
`--topics "topic1,topic2" --data data`? Or something else?